django-nonrel/ocl/collection/models.py

```diff
@@ -57,6 +57,9 @@ def get_version_model(cls):
     def clean(self):
         errors = {}
 
+        expressions_without_version = [self.drop_version(expression) for expression in self.expressions]
```

Comment:
this could be
```python
expressions_without_version = [self.drop_version(expression) for expression in self.expressions if 'mappings' in expression.url]
```